packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -1,75 +1,93 @@
 .content {
   z-index: var(--elevation-tooltip);
   max-width: var(--popover--width);
+  min-width: calc(var(--space-extravagant) * 3.75);
   box-shadow: var(--shadow-base);
+  margin-top: var(--space-small);
   border: var(--border-base) solid var(--color-border);
-  border-radius: var(--radius-base);
-  font-size: var(--typography--fontSize-base);
-  line-height: normal;
+  border-radius: var(--radius-larger);
+  overflow: scroll;
+  overflow-y: auto;
   background: var(--color-surface);
 }
 
 /* CAUTION: Tests for content visibility assume this class' presence hides the content
-  * so please be careful if you change this class name, or the css rules within.
- */
+* so please be careful if you change this class name, or the css rules within.
+*/
 .content.hidden {
   display: none;
   visibility: hidden;
 }
 
+.clearSearch {
+  display: flex;
+  position: absolute;
+  top: 50%;
+  right: var(--space-base);
+  z-index: var(--elevation-tooltip);
+  width: calc(var(--space-small) * 3);
+  height: calc(var(--space-small) * 3);
+  padding: 0;
+  border: none;
+  border-radius: var(--radius-circle);
+  background-color: var(--color-surface--background);
+  transform: translateY(-50%);
+  justify-content: center;
+  align-items: center;
+}
+
 .optionsList {
-  max-height: 300px;
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-smaller);
+  max-height: calc(var(--space-small) * 32.5);
   margin: 0;
-  padding: 0;
+  padding: var(--space-small);
   overflow: scroll;
   list-style: none;
 }
 
 .option {
-  padding: var(--space-small) var(--space-base);
+  display: flex;
+  min-height: calc((var(--space-largest) - var(--space-smaller)));
+  box-sizing: border-box;
+  padding: var(--space-small);
+  color: var(--color-heading);
+  font-weight: 500;
   cursor: pointer;
+  justify-content: space-between;
+  align-items: center;
 }
 
-.option:hover {
+.selectedOption,
+.option:hover,
+.option:focus {
+  border-radius: var(--radius-large);
```

Comment:
Same as in the actions, 

- is there a reason not to set the border-radius on the `option`, rather than modifying it based on state?
- should have the same transition property applied